.github/workflows/release.yml

```diff
@@ -24,6 +24,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Set up OpenJDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'adopt'
```

Comment:
Any reason we are using different distribution under [ci-build.yml](https://github.com/oasisprotocol/oasis-wallet-web/pull/1944/files#diff-68ba9d12f6f2601fe3c9f7f9d0d02aaea52384559d6d08fae7afc941663fdfcd)

Reply:
forgot to update